frontend/src/metabase/containers/dnd/ItemsDragLayer.jsx

```diff
@@ -120,9 +119,11 @@ class DraggedItems extends Component {
       >
         <BaseItemsTable
           items={items}
-          renderItem={this.renderItem}
+          ItemComponent={props => this.renderItem(props)}
```

Comment:
why is this needed?

Reply:
It seemed more conventional, and potentially less bug-prone, to pass a component rather than a function